src/ui/src/js/run.js

```diff
@@ -87,8 +87,11 @@ export default function appRun(
         // coalescePermissions [0] is roles, [1] is permissions
         user.permissions = RoleService.coalescePermissions(user.roles)[1];
 
-        let theme = _.get(user, 'preferences.theme', 'default');
-        $rootScope.changeTheme(theme);
+        // If user is logged in, change to their default theme selection
+        if (user.id != null){
```

Comment:
Is this to exclude the anonymous user?

Reply:
Yes, if the user is anonymous then use what is cached in their browser storage. If not, then use what we store with that user account.